interface/tuntap/windows/TAPDevice.c

```diff
@@ -233,46 +242,81 @@ static int get_device_guid(
         }
 
         if (is_tap_win32_dev(enum_name)) {
-            snprintf(name, name_size, "%s", enum_name);
-            if (actual_name) {
-                if (CString_strcmp(actual_name, "") != 0) {
-                    if (CString_strcmp(name_data, actual_name) != 0) {
-                        RegCloseKey (connKey);
-                        ++i;
-                        continue;
-                    }
-                }
-                else {
-                    snprintf(actual_name, actual_name_size, "%s", name_data);
-                }
+            struct Taps* tap = Allocator_calloc(alloc, sizeof(struct Taps), 1);
+            Bits_memcpy(tap->guid, enum_name, sizeof(enum_name));
+            Bits_memcpy(tap->name, name_data, sizeof(name_data));
+            if (!taps) {
+                taps = tap;
+                taps->next = NULL;
             }
-            stop = 1;
+            if (tail) {
+                tail->next = tap;
+            }
+            tail = tap;
         }
 
         RegCloseKey(connKey);
+        ++i;
     }
 
     RegCloseKey (control_net_key);
 
-    if (stop == 0) {
-        return -1;
+    return taps;
+}
+
+static int get_device_guid(
+    char *name,
```

Comment:
General rule of formatting arguments to leave first argument in first row and align rest to it.